src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,65 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private static final String COMMA = ",";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String reportData = readFromFile(fromFileName);
+        String report = createReport(reportData);
+        writeToFile(report, toFileName);
+    }
+
+    private String readFromFile(String fromFileName) {
```

Comment:
In **readFromFile** method you should.... read from file!)))
Just return the file content. Do not do any extra work - it is unexpected.
Move calculation to **createReport**